src/unix/linux_like/android/mod.rs

```diff
@@ -528,16 +526,16 @@ s_no_extra_traits! {
     }
 
     pub struct dirent {
-        pub d_ino: u64,
-        pub d_off: i64,
+        pub d_ino: crate::ino_t,
+        pub d_off: crate::off_t,
         pub d_reclen: c_ushort,
         pub d_type: c_uchar,
         pub d_name: [c_char; 256],
     }
 
     pub struct dirent64 {
-        pub d_ino: u64,
-        pub d_off: i64,
+        pub d_ino: crate::ino64_t,
```

Comment:
Same comment applies here - `dirent64` and `dirent` [are the same struct](https://cs.android.com/android/platform/superproject/main/+/main:bionic/libc/include/dirent.h;l=74-77?q=d_ino&ss=android%2Fplatform%2Fsuperproject%2Fmain:bionic%2Flibc%2Finclude%2F&start=1) on Android.